packages/runtime-common/matrix-constants.ts

```diff
@@ -25,7 +25,7 @@ export const APP_BOXEL_HAS_CONTINUATION_CONTENT_KEY =
   'app.boxel.has-continuation';
 export const APP_BOXEL_CONTINUATION_OF_CONTENT_KEY =
   'app.boxel.continuation-of';
-export const DEFAULT_LLM = 'openai/gpt-4.1';
+export const DEFAULT_LLM = 'openai/gpt-4.1-nano';
```

Comment:
You are changing the DEFAULT_LLM for the whole app. I DO NOT think this is the intended behaviour at all. We just want nano to be run for remix action right